crates/bevy_transform/src/components/global_transform.rs

```diff
@@ -9,33 +9,62 @@ use bevy_reflect::{std_traits::ReflectDefault, Reflect};
+///   is usually to consider `1.0` equivalent to 1 meter.
+/// - The `rotation` of an entity in space, [`Quat::IDENTITY`] representing the rotation
+///   where [`Vec3::Z`] is forward and [`Vec3::Y`] is up.
+/// - The `scale` of an entity, `1.0` representing the "size values" of the entity
+///   matching the main reference frame coordinates.
+///   It can be seen as a ratio of the main reference frame length unit to
+///   the entity's length unit.
+///   For example if the scale is `1.0` for a `Mesh`, its vertices position attribute
+///   have the same unit than the values used inside `translation`.
 ///
 /// ## [`Transform`] and [`GlobalTransform`]
 ///
-/// [`Transform`] transforms an entity relative to its parent's reference frame, or relative to world space coordinates,
-/// if it doesn't have a [`Parent`](bevy_hierarchy::Parent).
+/// [`Transform`] is the local transform of an entity in space relative to its parent transform,
+/// or the global transform relative to the main reference frame if it doesn't have a [`Parent`].
 ///
-/// [`GlobalTransform`] is managed by Bevy; it is computed by successively applying the [`Transform`] of each ancestor
-/// entity which has a Transform. This is done automatically by Bevy-internal systems in the system set
-/// [`TransformPropagate`](crate::TransformSystem::TransformPropagate).
+/// [`GlobalTransform`] is the absolute transform of an entity in space, relative to the main reference frame.
 ///
-/// This system runs during [`PostUpdate`](bevy_app::PostUpdate). If you
-/// update the [`Transform`] of an entity in this schedule or after, you will notice a 1 frame lag
-/// before the [`GlobalTransform`] is updated.
+/// [`GlobalTransform`] is updated from [`Transform`] by systems in the system set
+/// [`TransformPropagate`].
+/// Those systems run during [`PostUpdate`].
+/// If you update the [`Transform`] of an entity in this schedule or after,
+/// you may notice a 1 frame lag before the [`GlobalTransform`] is updated.
 ///
 /// # Examples
 ///
-/// - [`transform`][transform_example]
+/// - The [`transform example`], or the other examples in the [`transforms folder`],
+///   to learn how to use the [`Transform`] of an entity.
+/// - The [`parenting example`], to learn how [`Transform`] behaves in a hierarchy.
 ///
-/// [transform_example]: https://github.com/bevyengine/bevy/blob/latest/examples/transforms/transform.rs
+/// [`compute_transform`]: GlobalTransform::compute_transform
+/// [`TransformBundle`]: crate::TransformBundle
+/// [`TransformPropagate`]: crate::TransformSystem::TransformPropagate
+/// [`PostUpdate`]: bevy_app::PostUpdate
+/// [`Parent`]: bevy_hierarchy::Parent
+/// [`transform example`]: https://github.com/bevyengine/bevy/blob/latest/examples/transforms/transform.rs
+/// [`transforms folder`]: https://github.com/bevyengine/bevy/tree/latest/examples/transforms
+/// [`parenting example`]: https://bevyengine.org/examples/3d/parenting/
```

Comment:
@rparrett originally asked that we use either GitHub or bevyengine.org links for examples. I have the same question, but I've also heard folks suggest maybe don't link at all. Anyone want to weigh in on a standard practice for example docs?

Reply:
The `bevyengine.org` link is better. A real fix wants https://github.com/rust-lang/rust/issues/130493